packages/pn-personagiuridica-webapp/src/components/Contacts/AddSpecialContactDialog.tsx

```diff
@@ -97,18 +99,7 @@ const AddSpecialContactDialog: React.FC<Props> = ({
     await formik.setFieldTouched('sender', true, false);
     await formik.setFieldValue('sender', { id: newValue?.id ?? '', name: newValue?.name ?? '' });
 
-    if (
-      !(
-        formik.values.channelType === ChannelType.SERCQ_SEND &&
-        pecInValidationForSender(newValue?.id ?? '')
-      )
-    ) {
-      setIsPecInValidation(false);
-    } else {
-      setIsPecInValidation(true);
-    }
-
-    if (newValue && addressesData.addresses.some((a) => a.senderId === newValue.id)) {
+    if (newValue && addressesData.addresses.some((a) => a.senderId === newValue.id) && formik.values.channelType) {
```

Comment:
Here you missed a code from develop that mustn't be removed

Reply:
done